Deploy/dependenciesscanner.cpp

```diff
@@ -28,8 +28,8 @@ PrivateScaner DependenciesScanner::getScaner(const QString &lib) const {
 
     auto sufix = info.completeSuffix();
 
-    if (sufix.compare("dll", Qt::CaseSensitive) == 0 ||
-            sufix.compare("exe", Qt::CaseSensitive) == 0) {
+    if (sufix.contains("dll", Qt::CaseSensitive) == 0 ||
+            sufix.contains("exe", Qt::CaseSensitive) == 0) {
```

Comment:
```suggestion
    if (sufix.contains("dll", Qt::CaseSensitive)  ||
            sufix.contains("exe", Qt::CaseSensitive)) {
```
I think you forget to remove == 0 because the contains method return boolean value but not int.